specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-06-02-preview/managedClusters.json

```diff
@@ -6712,6 +6722,20 @@
         }
       }
     },
+    "UlimitConfig": {
+      "description": "Ulimit settings for Linux agent nodes",
+      "type": "object",
+      "properties": {
+        "maxLockedMemory": {
+          "type": "string",
+          "description": "Maximum locked-in-memory address space (KB)"
+        },
+        "noFile": {
```

Comment:
`no` is the actual abbreviation the kernel uses is that right, and we're just matching it here?

Reply:
Yes, `no` is used in kernel, an example here: https://access.redhat.com/solutions/61334